go.mod

```diff
@@ -1,3 +1,8 @@
 module github.com/plutov/paypal/v3
 
 go 1.12
+
+require (
+	github.com/brianvoe/gofakeit v3.18.0+incompatible
```

Comment:
I don't think we should add third-party package just to generate UUID, I'd rather write some small util func.